build.sh

```diff
@@ -781,10 +781,10 @@ fi
 if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
     if [ $__CrossBuild == 1 ]; then
         __ClangMajorVersion=3
-        __ClangMinorVersion=6
+        __ClangMinorVersion=9
     else
         __ClangMajorVersion=3
-        __ClangMinorVersion=5
+        __ClangMinorVersion=9
```

Comment:
Is it normal that both "if" branches are equal? I think it would be good to delete inner condition "if [ $__CrossBuild == 1 ];"